assembly/hdp2.6.3/src/main/resources/common-services/SPLICEMACHINE/2.5.1/service_advisor.py

```diff
@@ -277,10 +262,10 @@ def getMasterDashDProperties(self):
         "splice.spark.yarn.am.extraLibraryPath=/usr/hdp/current/hadoop-client/lib/native",
         "splice.spark.yarn.am.waitTime=10s",
         "splice.spark.yarn.executor.memoryOverhead=2048",
-        "splice.spark.driver.extraJavaOptions=-Dlog4j.configuration=file:/etc/spark/conf/log4j.properties",
+        "splice.spark.yarn.am.extraJavaOptions=-Dhdp.version=2.6.3.0-235",
```

Comment:
do we need to provide precise version to the build level?  will be issue if somebody tried different version of 2.6.3.0?

Reply:
Yep and that would be a good thing since we have not tested it.